mitmproxy/net/tls.py

```diff
@@ -5,7 +5,7 @@
 from enum import Enum
 from functools import lru_cache
 from pathlib import Path
-from typing import Iterable, Callable, Optional, Tuple, List, Any, BinaryIO
+from typing import Iterable, Callable, Optional, Tuple, List, Any, BinaryIO, List
```

Comment:
```suggestion
from typing import Iterable, Callable, Optional, Tuple, List, Any, BinaryIO
```

List is already imported.